sale_commission/model/partner.py

```diff
@@ -2,7 +2,8 @@
 ##############################################################################
 #
 #    OpenERP, Open Source Management Solution
-#    Copyright (C) 2011 Pexego Sistemas Informáticos (<http://www.pexego.es>). All Rights Reserved
+#    Copyright (C) 2011 Pexego Sistemas Informáticos (<http://www.pexego.es>).
+#    All Rights Reserved
```

Comment:
You can remove 'All Rights Reserved' it is obsolete since 2000
And the $Id$ too